### ISSUE TITLE
failed to open IP:8787

### ISSUE THREAD

**Issue by vebaev** [rocker-org/rocker-versioned2#572]
I'm running on my server the docker image with:
`sudo docker run --rm -p 8787:8787 -d -e PASSWORD=xxxxxxxxx rocker/rstudio`

but failed to open IP:8787

The image was running fine months, but I needed to delet it and download again yesterday and uppon runing cannot connect. 

PS my other dockers are running fine and connecting to them is fine

**Comment by eitsupi** [comment]
What does "failed to open IP:8787" mean?
What does the following command show in the terminal? (Display logs in the terminal with the `-i` and `-t` options)

```sh
sudo docker run --rm -it -p 8787:8787 -d -e PASSWORD=xxxxxxxxx rocker/rstudio
```

**Comment by vebaev** [comment]
Fail to open I mean I failed to connect via the browser on port 8787. But running with -ti option I see it is stuck on:

`sudo docker run --rm -ti -p 8787:8787 -e ROOT=TRUE -e DISABLE_AUTH=TRUE -e PASSWORD=xxxxxxxx rocker/rstudio
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 01_set_env: executing... 
skipping /var/run/s6/container_environment/HOME
skipping /var/run/s6/container_environment/PASSWORD
skipping /var/run/s6/container_environment/RSTUDIO_VERSION
[cont-init.d] 01_set_env: exited 0.
[cont-init.d] 02_userconf: executing... 
Skipping authentication as requested
Adding user `rstudio' to group `sudo' ...
Adding user rstudio to group sudo
Done.
rstudio added to sudoers
[cont-init.d] 02_userconf: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
TTY detected. Printing informational message about logging configuration. Logging configuration loaded from '/etc/rstudio/logging.conf'. Logging to ''.
rserver[257]: ERROR Unable to determine install path: system error 1 (Operation not permitted); LOGGED FROM: virtual rstudio::core::ProgramStatus rstudio::server::Options::read(int, char* const*, std::ostream&) src/cpp/server/ServerOptions.cpp:147

2022-12-08T13:36:10.242757Z [rserver] ERROR Unable to determine install path: system error 1 (Operation not permitted); LOGGED FROM: virtual rstudio::core::ProgramStatus rstudio::server::Options::read(int, char* const*, std::ostream&) src/cpp/server/ServerOptions.cpp:147
`

**Comment by eitsupi** [comment]
What happens if I run it without any extra options? e.g. `docker run --rm -it -p 8787:8787 rocker/rstudio`
What is your OS and Docker version?

**Comment by vebaev** [comment]
I got same error. I use:
centos-release-7-7.1908.0.el7.centos.x86_64
Client: Docker Engine - Community Version: 19.03.8

**Comment by eitsupi** [comment]
> centos-release-7-7.1908.0.el7.centos.x86_64
> Client: Docker Engine - Community Version: 19.03.8

I think the Docker version is too old.
Please update to the latest version and try again.

**Comment by eitsupi** [comment]
> What is the difference between rocker/rstudio:4.0.5-daily and rocker/rstudio:4.0.5 ?

`rocker/rstudio:4.0.5-daily` is an old image, some years old. There are no vulnerability fixes; `rocker/rstudio:4.0.5` is a recent build.

The problem is probably caused by old host OS kernel or Docker Engine.
There is nothing we can do, so I close it.
